docs/windows/attributes/cpp-attributes-com-net.md

```diff
@@ -96,11 +95,11 @@ This field lists other attributes that are incompatible with the specified attri
 
 ## Debug injected code
 
-Using attributes can greatly simplify C++ programming. For more information, see [Concepts](/cpp/windows/attributed-programming-concepts). Some attributes are interpreted directly by the compiler. Other attributes inject code into the program source, which the compiler then compiles. This injected code makes programming easier by reducing the amount of code you have to write. Sometimes, however, a bug may cause your application to fail while it is executing injected code. When this happens, you will probably want to look at the injected code. Visual Studio provides two ways for you to see injected code:
```

Comment:
The "For more information" sentence is removed as the link just redirects back to this topic:

https://github.com/MicrosoftDocs/cpp-docs/blob/2643b99aa3b52c23f8c06438689160b57404aa25/.openpublishing.redirection.json#L11044-L11045